install_scripts/templates/replicated-operator.sh

```diff
@@ -377,6 +380,10 @@ if [ "$NO_PROXY" != "1" ]; then
     if [ -z "$PROXY_ADDRESS" ]; then
         promptForProxy
     fi
+
+    if [ -n "$PROXY_ADDRESS" ]; then
+        getNoProxyAddresses "$DAEMON_ENDPOINT"
```

Comment:
should you add private address here as well?

Reply:
On the master the daemon endpoint is the private address and on workers I couldn't think of any reason to add the private address. I guess one possibility would be if someone is running their own private registry in their app, but in that case we'd need to maintain a full list of all IPs in the cluster.